cranelift/codegen/src/prelude.isle

```diff
@@ -307,18 +344,18 @@
 
 (type SideEffectNoResult (enum (Inst (inst MInst))))
 
-;; Create an empty `ValueRegs`, but do emit the given side-effectful
+;; Create an empty `InstOutput`, but do emit the given side-effectful
 ;; instruction.
-(decl value_regs_none (SideEffectNoResult) ValueRegs)
-(rule (value_regs_none (SideEffectNoResult.Inst inst))
+(decl sideeffect (SideEffectNoResult) InstOutput)
+(rule (sideeffect (SideEffectNoResult.Inst inst))
```

Comment:
nitpick: Could we call this `side_effect`? This would match how we have the `SideEffect` type (not `Sideeffect`) right now.

Reply:
Right, this makes sense.